engine/dumps/bank03.asm

```diff
@@ -2603,50 +2603,50 @@ Datae4e7:
 	db $03, $d8, $af
 	db $03, $20, $b5
 
-Functione505:
-	ld hl, Datae5a5
+BoxEditMenu:
+	ld hl, BoxEditMenuList
 	call LoadMenuHeader
 	call VerticalMenu
 	call CloseWindow
 	ret c
 	ld a, [w2DMenuDataEnd]
 	cp $01
-	jr z, Functione53e
+	jr z, WhenYouChangeBoxTextFunc
 	cp $02
-	jr z, Functione57e
+	jr z, BoxNameChangeFunc
 	and a
 	ret
 
-Functione51f:
-	ld hl, Texte529
+BoxChangeUnderDevTextFunc:
+	ld hl, BoxChangeUnderDevText
 	call MenuTextBox
 	call CloseWindow
 	ret
 
-Texte529:
-	text "バンクチェンジは"
-	next "かいはつちゅうです！"
+BoxChangeUnderDevText:
+	text "バンクチェンジは" ; "Box change is"
+	next "かいはつちゅうです！" ; "under development!"
 	prompt
 
-Functione53e:
-	ld hl, Texte551
+WhenYouChangeBoxTextFunc:
+	ld hl, WhenYouChangeBoxText
 	call MenuTextBox
 	call YesNoBox
 	call CloseWindow
 	ret c
-	jr Functione51f
+	jr BoxChangeUnderDevTextFunc
 
 Functione54d:
 	ld a, [wMenuSelection]
 	ret
 
-Texte551:
-	text "#　ボックスを　かえると"
-	line "どうじに　レポートが　かかれます"
-	para "<⋯⋯>　それでも　いいですか？"
+WhenYouChangeBoxText:
+	text "#　ボックスを　かえると" ; "When you change a box"
+	line "どうじに　レポートが　かかれます" ; "data will be saved."
+	para "<⋯⋯>　それでも　いいですか？" ; "Is that okay?"
 	done
 
-Functione57e:
+BoxNameChangeFunc:
```

Comment:
```suggestion
ChangeBoxName:
```

Since we append things like `Text`, `RLE`, `Data`, ect at the end of data labels... It doesn't add value to add `Func` at the end of Routines.